files/build_templates/sonic_debian_extension.j2

```diff
@@ -214,6 +214,12 @@ if [[ $CONFIGURED_ARCH == amd64 ]]; then
 sudo DEBIAN_FRONTEND=noninteractive dpkg --root=$FILESYSTEM_ROOT -i $debs_path/kdump-tools_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true chroot $FILESYSTEM_ROOT apt-get -q --no-install-suggests --no-install-recommends --force-no install
 fi
+# Install python-swss-common package and all its dependent packages
```

Comment:
Please add a newline above this comment to separate the sections.

Reply:
@jleveque Done